share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -44,10 +101,22 @@ function(rocm_install_targets)
         target_include_directories(${TARGET} INTERFACE $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>)
     endforeach()
 
+    if(PARSE_COMPONENT)
+        set(COMPONENT_ARG "COMPONENT;${PARSE_COMPONENT}")
+        set(LIB_COMPONENT_ARG "COMPONENT;${PARSE_COMPONENT}")
+    elseif(ROCM_USE_DEV_COMPONENT)
+        set(COMPONENT_ARG "COMPONENT;devel")
+        set(LIB_COMPONENT_ARG "NAMELINK_COMPONENT;devel")
```

Comment:
`NAMELINK_COMPONENT` is not an available option in ubuntu 18.04. You should check the cmake version is at least 3.12 otherwise just fallback on this being in the runtime component.

Reply:
Alternatively, the older method of splitting the namelink and library files (using `NAMELINK_ONLY` and `NAMELINK_SKIP`, which are supported at least back to CMake 3.0) is still supported in later versions of CMake, so it can be used for all platforms.